lib/module.js

```diff
@@ -99,6 +99,16 @@ export default function SentryModule (moduleOptions) {
 
   // Register the client plugin
   if (!options.disabled && !options.disableClientSide) {
+    const apmIntegrations = {}
+    const integrationsKeys = Object.keys(options.clientIntegrations)
+    integrationsKeys.forEach((integrationName) => {
+      const integration = options.clientIntegrations[integrationName]
+      if (integration.apm) {
+        apmIntegrations[integrationName] = integration
+        delete options.clientIntegrations[integrationName]
```

Comment:
We'd need this documented also but before we do, we should decide on the best way forward.

I think this module should make it super easy to enable tracing.

Right now one, per the example in documentation:

```js
import * as Sentry from "@sentry/browser";
import { Vue as VueIntegration } from "@sentry/integrations";
import { Integrations } from "@sentry/apm";
import Vue from "vue";

Sentry.init({
  // ...
  integrations: [
    new Integrations.Tracing(),
    new VueIntegration({
      Vue,
      tracing: true
    })
  ],
  tracesSampleRate: 1
});
```

has to do three things:
 - enable `tracing: true` for the Vue integration
 - add Tracing integration
 - set `tracesSampleRate` option.

All those can be done currently manually, I guess, but I think we should make it simpler. Possibly just a separate object option for this like:

```
enableTracing: {
   tracesSampleRate: 1,
   options: {}  // Optional options for the tracing integration
}
```

Reply:
@rchl I think it makes sense to use `enableTracing` option, because there are only two Integration inside APM - Tracing and Expess (which is not needed for nuxt). I think i can extend this PR for this option.

Reply:
I'm just having second thoughts about what will happen when they add another "performance monitoring" integration that we'll want to enable...

But it probably will still make sense to have tracing-specific configuration.